README.md

```
@@ -30,17 +30,46 @@
+
+It should be added to `env.json` as:
+```json
+{
+  "FIRESTORE_CREDENTIALS": "{\"type\":\"service_account\",\"project_id\":\"your-project-id\",\"private_key_id\":\"key-id\",\"private_key\":\"-----BEGIN PRIVATE KEY-----\\nkey-content\\n-----END PRIVATE KEY-----\\n\",\"client_email\":\"email@example.com\"}"
+}
+```
+
+## 3. Running the Server
+After completing the above steps, you can now run the server locally with your configured environment.
+
+##### *Executing the script to run the server locally*
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h5

(MD001, heading-increment)